tests/integration_tests/run_constant_reprocessing_serpent/test.py

```diff
@@ -29,8 +33,9 @@ def test_integration_2step_constant_ideal_removal_heavy(setup):
         cwd=cwd,
         stdout=sys.stdout,
         stderr=subprocess.STDOUT)
-    np.testing.assert_equal(read_keff(test_db), read_keff(ref_db))
-    assert_db_almost_equal(test_db, ref_db, tol)
+    np.testing.assert_allclose(read_keff(test_db)[0], read_keff(ref_db)[0], rtol=5e-2)
+    np.testing.assert_allclose(read_keff(test_db)[1], read_keff(ref_db)[1], rtol=5e-1)
```

Comment:
How come these don't use `tol` for `rtol`? Is `tol` here `atol`?

Reply:
keff requires a difference value of tolerance than the material compositions.